{"uuid": "87b9cff4-674c-4b1d-9155-7144b5f757df", "children": ["bc5f561b-165a-4b37-be8b-837a20c9e02f"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../tests/tests/config.json'\n", "trace": "  File \"/Users/yaroslav.petryk/Desktop/projects/powerfox-app-native-android/tests/venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/yaroslav.petryk/Desktop/projects/powerfox-app-native-android/tests/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/yaroslav.petryk/Desktop/projects/powerfox-app-native-android/tests/venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/yaroslav.petryk/Desktop/projects/powerfox-app-native-android/tests/tests/conftest.py\", line 11, in driver\n    locale = str(config_importer.config_locale())\n  File \"/Users/yaroslav.petryk/Desktop/projects/powerfox-app-native-android/tests/helpers/config_importer.py\", line 67, in config_locale\n    config = self.config()\n  File \"/Users/yaroslav.petryk/Desktop/projects/powerfox-app-native-android/tests/helpers/config_importer.py\", line 12, in config\n    with open(self.CONFIG_PATH, 'r') as config_file:\n"}, "start": 1683573591406, "stop": 1683573591406}], "start": 1683573591406, "stop": 1683573591488}